src/util/styles.ts

```diff
@@ -1,11 +1,13 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
-import { IFontWeight } from '@uifabric/styling/lib';
-import { ICSSPixelUnitRule } from '@uifabric/merge-styles/lib/IRawStyleBase';
+import { IRawStyle } from '@uifabric/styling';
 
-const borderRadiusSoft: ICSSPixelUnitRule = '2px';
-const fontWeightBold: IFontWeight = '600';
+export const borderRadiusSoft = '2px';
```

Comment:
what do you think about objects or enums for properties with known possible values? this might be more clear in the consuming components

```
const borderRadius = {
    soft: '2px',
    round: '50%',
};
const fontWeight = {
    normal: '400',
    bold: '600',
};
```

Reply:
Yeah, I went back and forth a bit. What's challenging about the way your proposing is that the styles of all the values get inferred as `string`, and as an example `IRawStyle` doesn't accept a `string` for fontWeight, so you'd have to cast or declare as `IFontWeight`. Which is doable, but felt more cumbersome.

Reply:
not a big deal, we can revisit if we think it'll help